zstd_fallback.go

```diff
@@ -0,0 +1,17 @@
+// +build !cgo
+
+package sarama
+
+import (
+	"errors"
+)
+
+var errZstdCgo = errors.New("zstd compression requires building with cgo enabled")
```

Comment:
Should this error be public?

Reply:
I considered that, but I didn't think it would be useful (what is the user going to do differently in this case?). If it were public, we would have to maintain it forever to keep backwards compatibility (e.g. even after switching to a pure-go zstd library).